executor/show_test.go

```diff
@@ -609,6 +610,23 @@ func (s *testSuite5) TestShowCreateTable(c *C) {
 			"  `a` varchar(123) DEFAULT NULL\n"+
 			") ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin",
 	))
+
+	configAllowAutoRand := &config.GetGlobalConfig().Experimental.AllowAutoRandom
+	originVal := *configAllowAutoRand
+	defer func() {
+		*configAllowAutoRand = originVal
+	}()
+	*configAllowAutoRand = true
+
+	tk.MustExec("create table auto_random_tbl (a bigint primary key auto_random(3), b varchar(255))")
```

Comment:
```
mysql> create table auto_random_tbl (a bigint primary key auto_random, b varchar(255));
OK
mysql> show create table auto_random_tbl;
| auto_random_tbl | CREATE TABLE `auto_random_tbl` (
  `a` bigint(20) NOT NULL /*T!40000 AUTO_RANDOM(5) */,
  `b` varchar(255) DEFAULT NULL,
  PRIMARY KEY (`a`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin |
```
I think it's fine to gives the explicit shard_length `5` for `SHOW CREATE TABLE`. Please confirm this.

Reply:
> Omitting this value is equivalent to setting it to 5. In other words, auto_random has the same meaning as auto_random(5).

I think it is consistent with the PR description. We will mention it in the documentation.